Realm/RLMObject.mm

```diff
@@ -40,7 +40,11 @@ - (instancetype)init {
 }
 
 - (instancetype)initWithValue:(id)value {
-    return [super initWithValue:value schema:RLMSchema.sharedSchema];
+    RLMSchema *schema = RLMSchema.partialSharedSchema;
+    if (![schema schemaForClassName:self.class.className]) {
```

Comment:
right, to support standalone objects, we need to use the shared schema, which leads me to my next question: why not keep this code as-is, if we're going to default to the shared schema anyway?